webview/src/experiments/components/table/body/commitsAndBranches/CommitsAndBranchesNavigation.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { AddAndRemoveBranches } from './AddAndRemoveBranches'
+import styles from './styles.module.scss'
+import {
+  showLessCommits,
+  showMoreCommits,
+  switchToBranchesView,
+  switchToCommitsView
+} from '../../../../util/messages'
+import tableStyles from '../../styles.module.scss'
+import { ExperimentsState } from '../../../../store'
+
+export const CommitsAndBranchesNavigation: React.FC = () => {
+  const { hasMoreCommits, isBranchesView, isShowingMoreCommits } = useSelector(
+    (state: ExperimentsState) => state.tableData
+  )
+
+  return (
+    <tbody>
+      <tr>
+        <td className={tableStyles.experimentsTd}>
+          <div className={styles.commitsAndBranchesNav}>
+            {hasMoreCommits && (
+              <button
+                className={styles.commitsAndBranchesNavButton}
+                onClick={showMoreCommits}
+                disabled={isBranchesView}
+              >
+                Show More Commits
+              </button>
+            )}
+            {isShowingMoreCommits && (
+              <button
+                className={styles.commitsAndBranchesNavButton}
+                onClick={showLessCommits}
+                disabled={isBranchesView}
+              >
+                Show Less Commits
+              </button>
+            )}
+
+            <AddAndRemoveBranches />
+
+            <span className={styles.separator} />
+
+            <button
+              className={styles.commitsAndBranchesNavButton}
+              onClick={
+                isBranchesView ? switchToCommitsView : switchToBranchesView
+              }
+            >
+              {isBranchesView
```

Comment:
[Q] Do we need to/should we keep this around long term?

Reply:
I think people in general feel like this won't be necessary anymore, I don't mind removing it once the whole feature is in. That being said, I personally feel like this could be useful before selecting branches to add as it gives the user an overview of what is in each branch. Also if you do not particularly case about previous commits and have too many branches to easily select, this is a quick way to view everything at once. 

This isn't too complicated to remove, nor it is to keep it. The most problematic part about keeping it is where to show it now that we have controls on every branch. I have no preferences about it.

Reply:
Trying to limit/fight complexity.